net/c-ares/Portfile

```diff
@@ -1,26 +1,32 @@
-PortSystem 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+PortGroup        cmake 1.1
 
 name             c-ares
 version          1.16.1
+revision         1
+platforms        darwin
+license          MIT
 categories       net
 maintainers      {toby @tobypeterson}
+
 description      asynchronous DNS library
-long_description \
-    c-ares is a C library that performs DNS requests and \
-    name resolves asynchronously. c-ares is a fork of the \
-    library named 'ares', written by Greg Hudson at MIT.
+long_description c-ares is a C library that performs DNS requests and \
+                 name resolves asynchronously. c-ares is a fork of the \
+                 library named 'ares', written by Greg Hudson at MIT.
+
 homepage         https://c-ares.haxx.se/
-platforms        darwin
-license          MIT
+
 master_sites     ${homepage}download/
 checksums        rmd160 31e7f455f8e700f1879877a1f3d841b1ca395b90 \
                  sha256 d08312d0ecc3bd48eee0a4cc0d2137c9f194e0a28de2028928c0f6cae85f86ce \
                  size   1374637
 
 patchfiles       patch-ares_init.c.diff
 
-configure.args   --mandir=${prefix}/share/man \
-                 --enable-shared
+#configure.args   --mandir=${prefix}/share/man \
+#                 --enable-shared
```

Comment:
Why leave these commented-out lines in the port?

Reply:
Oops thanks.